utils/generate-domains-blocklists/generate-domains-blocklist.py

```diff
@@ -278,6 +278,11 @@ def blocklists_from_config_file(
 )
 argp.add_argument(
     "-w",
+    "--whitelist",
```

Comment:
I don't think we really need to add a new long option for an option which is deprecated.

Reply:
Ah wait, I see, this is a follow up to #1406 I see now 👍